projects/tempest-finance/index.js

```diff
@@ -0,0 +1,20 @@
+const chains = require("./chains");
+const {sumTokens} = require("../helper/sumTokens");
+
+async function tvl(api, chain) {
+    let vaults = chains[chain];
+    const tokens = vaults.map(vault => vault.asset);
+    const balances = await api.multiCall({ abi: 'uint256:totalAssets', calls: vaults.map(vault => vault.address) });
```

Comment:
curious, how does this work? since you are LPing into ambient, there are two tokens right?

Reply:
basically on our vault, we have a main asset index (which is one of the 2 tokens) and all the price is calculated in that main asset.

The `totalAssets` function uses oracle to calculate the price the other token in the main asset, and normalize the total asset in main asset.